develop-docs/development/environment/index.mdx

```diff
@@ -73,7 +71,9 @@ Running `sentry devserver` will automatically use HTTPS when the certificates ha
 
 ### Ingestion Pipeline (Relay) aka Sending Events to your Dev Environment
 
-<Link to="/services/relay/">Relay</Link> and the ingest workers are not started by default. Follow the instructions below to start them so you can send events to your dev environment Sentry instance:
+<Link to="/services/relay/">Relay</Link> and the ingest workers are not started by
+default. Follow the instructions below to start them so you can send events to your
+dev environment Sentry instance:
```

Comment:
any reasons why you broke the line? it would be nice to be consistent

Reply:
Formatter... 🙈 I will revert this change